index.html

```diff
@@ -6200,6 +6200,11 @@ <h4>Plain HTML or Polyfill DOM Result of the MathML Quadratic Formula</h4>
             </p>
             <p>To be <a>keyboard accessible</a>, authors SHOULD manage focus of descendants for all instances of this <a>role</a>, as described in <a href="#managingfocus">Managing Focus</a>.</p>
             <p>Elements with the role <code>menu</code> have an implicit <pref>aria-orientation</pref> value of <code>vertical</code>.</p>
+            <p class="note">
+              The <code>menu</code> <a>role</a> doesn't allow for accessibility children roles other than <rref>menuitem</rref>, <rref>menuitemcheckbox</rref>, <rref>menuitemradio</rref>,
+              <rref>separator</rref>, and <rref>group</rref>. In this context, the role <rref>group</rref> allows only <a>accessibility children</a> with a role <rref>group</rref>,
```

Comment:
The group role has 

> [...] when a group is used in the context of a listbox, for example, authors MUST limit its children to option elements.

I think this requirement here should be added there, too.

Reply:
Thanks @pkra 
Since it only contained an example, I initially didn't add this one to the group, but it’s probably worth including. I’ve added the menu and menubar scenario for better clarity as per your suggestion.